aws/usageReports/ec2/mappings.go

```diff
@@ -43,9 +43,6 @@ const TemplateLineItem = `
 {
```

Comment:
Your mapping isn't well readable, it should be:
- cpu
  - peak
  - average
- network
  - in
  - out....